src/main/resources/evolutions/default/12.sql

```diff
@@ -0,0 +1,39 @@
+# --- !Ups
+
+delete from "section";
+
+INSERT INTO section (id, section_number, opportunity_id, title, text) VALUES (1, 1, 1, 'About this opportunity', 'We want to achieve the widest benefit to society and the economy from the research we fund.
+
+This may be by sharing knowledge, commercialising ideas, exploring social benefits or other ways to increase the impact of your research.
+
+Under the Exploring Innovation Seminars programme, we will pay up to £2,000 for each event promoting innovation and collaboration. We will not pay for food or drink.
+
+Only organisations which receive funding from UK Research Councils may apply.
+
+As part of this, we want to help you develop innovative ways of building on the research you carry out.');
+INSERT INTO section (id, section_number, opportunity_id, title, text) VALUES (2, 2, 1, 'What we will ask you', '### Event title
+What is your event called? Wordcount: 20
```

Comment:
Yeah. Laika has some good support for customisation of the html. See http://planet42.github.io/Laika/customizing-laika/customize-rendering.html for some examples.